go-backend/service/response.go

```diff
@@ -0,0 +1,31 @@
+package service
+
+import (
+	"encoding/json"
+	"net/http"
+
+	logger "github.com/sirupsen/logrus"
+)
+
+type messageObject struct {
```

Comment:
can you confirm if some of these structs are not already added in some existing PRs which are merged now.

Reply:
As these structs are used for API's response, we think it should be present in the service package only.

There is one existing struct but it is defined in db package(organization.go)